test/integration/ruby_uninstall/controls/ruby_uninstall.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+system_ruby = '2.5.1'
+user_ruby = '2.5.0'
+
+control 'Global Ruby uninstall' do
+  title 'Global Ruby should be uninstalled'
+
+  desc "#{system_ruby} should be uninstalled"
+  describe bash('source /etc/profile.d/rbenv.sh && rbenv versions --bare') do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not match(/#{Regexp.quote(system_ruby)}/) }
```

Comment:
I cannot get this test to fail, I'm not sure how to access the system wide `rbenv` install.

As `vagrant` I only see the 2.5.0 version:
```console
$ rbenv versions --bare
2.5.0
```

As `root`, I see no Ruby version, although it is installed:
```console
$ ls -halt /usr/local/rbenv/versions/
total 12K
drwxr-xr-x 12 root root 4.0K Aug 20 12:34 ..
drwxr-xr-x  6 root root 4.0K Aug 20 12:34 2.5.1
drwxr-xr-x  3 root root 4.0K Aug 20 12:28 .
```

Reply:
Maybe it's because mixing system wide install and user install is not a good idea and [discouraged](https://github.com/rbenv/rbenv/issues/38#issuecomment-148413842)